pages/login.vue

```diff
@@ -62,8 +79,12 @@ async function submit() {
           type="password"
           class="form-control"
           placeholder="Enter Password"
+          @blur="v$.password.$touch"
         />
       </div>
+      <div v-for="error of v$.password.$errors" :key="error.$uid" class="input-errors">
+        <div class="error-msg">{{ error.$message }}</div>
+      </div>
       <button type="submit" class="btn btn-primary" :disabled="isLoading" html-type="submit" @click.prevent="submit">
```

Comment:
Users can submit the empty forms during create/edit user. It's not expected. have to add validation when trying to submit empty form.